### PR TITLE
Add readthedocs glue.

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Comment by matz-e** [comment]
Documentation build from the PR: https://brain-indexer.readthedocs.io/en/rtfd.io/intro.html

**Comment by 1uc** [comment]
There's a couple of environment variables that haven't been modernized (possibly just in the documentation):
https://brain-indexer.readthedocs.io/en/rtfd.io/developer_guide.html

We should upstream the Spack recipe and update the first option:
https://brain-indexer.readthedocs.io/en/rtfd.io/installation.html
 
... and devpi when we publish on pypi.

Very nice! It seems to all render reasonably well AFAICT.

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Jinja2~=3.0.0
+sphinx<5.1.0,>=5.0.0
```
Comment (1uc) [inline docs/requirements.txt:2]: Could be a first issue: make compatible with modern sphinx :)

**Review by 1uc — APPROVED** [review]

File: docs/requirements.txt (hunk 1 above)
Comment (matz-e) [reply docs/requirements.txt:2]: Yes, there's more with the Markdown conversion and following version pins, too :)

**Pull request merged**